src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.main.banner-mode=off
+
+spring.jpa.hibernate.ddl-auto=none
+
+spring.sql.init.mode=always
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/test
```

Comment:
is there a possibility to specify the database url itself (in this case postgresql://localhost:5432)?
Also I thnik the table should be created automatically to avoid the need of preconfiguring the database manually.

Reply:
see above

Reply:
spring.sql.init.mode=always should create the table automatically